fuzzing.py

```diff
@@ -0,0 +1,116 @@
+import mutations
+import glob 
+import subprocess
+import os
+import hashlib
+import random
+import time
+import threading
+
+class SimpleFuzzer:
+    def __init__(self, sample_dir_path, crashes_dir_path):
+        self.sample_dir_path = sample_dir_path
+        self.crashes_dir_path = crashes_dir_path
+        self.samples = self.load_samples()
+        self.amount_of_fuzzings = 0
+
+    '''
+    load all the samples into a data structure, without duplications.
+    there may be two diffrent files with same content, address that.
+    '''
+    def load_samples(self):
+        corpus_filenames = glob.glob(self.sample_dir_path + "/*")
+        corpus = set()
+        sample_content_dict = {}
+        for filename in corpus_filenames:
+            previous_corpus_size = len(corpus)
+            file_content = open(filename, 'rb').read()
+            corpus.add(file_content)
+            if len(corpus) != previous_corpus_size:
+                sample_content_dict[filename] = file_content 
+        return sample_content_dict
+
+    ''' 
+    create a new file with content and run it on the target_command_line 
+    target command line args is in the format: [[target], [-args]].
+    '''
+    def fuzz(self, file_save_fuzz_content, content, target_command_line_args, sample_name):
```

Comment:
This is not very good name, this function run the fuzzed file, not fuzzing herself